### PR TITLE
DOC-6397-Sync-Gateway-missing stats

### PR DESCRIPTION
**Pull request by ibsoln**
https://issues.couchbase.com/browse/DOC-6397

- Amended per feedback
- added-rev-cache-to-index
- Also add section headers into page
- Added another-tweak-to-links

### CODE REVIEW

**Review by ibsoln — COMMENTED** [review]
Per Jacques https://couchbase.slack.com/archives/D010WQHS4UC/p1586193100003600

**Comment by ibsoln** [comment]
Ready to merge need an and-1 approver

File: modules/ROOT/pages/stats-monitoring.adoc [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+Sync Gateway makes capturing that information easy, through its Admin REST API, which provides access to a number of useful metrics covering a node's:
 
-The following `curl` command requests the monitoring stats on the Admin REST API.
+* Performance
+* Resource utilization and
+* Health
+
+You can request the monitoring stats, from the Admin REST API, using the `expvars` method, as shown in this `curl` command:
 
 [source,java]
```
Comment (simon-dew) [inline modules/ROOT/pages/stats-monitoring.adoc]: Should be [source,console]

File: modules/ROOT/pages/stats-monitoring.adoc [hunk 2]
```diff
@@ -117,113 +183,130 @@ The response is a JSON object with the following schema.
...
 
-===== process_cpu_percent_utilization
+==== process_cpu_percent_utilization
 
-CPU utilization (%).
+Description::
+The CPU's utilization as percentage value.
+Derivation::
+The CPU usage calculation is performed based on _user_ and _system_ CPU time, but it doesn't include components such as `iowait`.
+This means that the values of `process_cpu_percent_utilization` and `%Cpu`, returned when running the `top` command, will *differ*  .
```
Comment (simon-dew) [inline modules/ROOT/pages/stats-monitoring.adoc]: Delete space after `*differ*` and before full stop.

File: modules/ROOT/pages/stats-monitoring.adoc [hunk 3]
```diff
@@ -117,113 +183,130 @@ The response is a JSON object with the following schema.
...
+Description::
 Number of warnings logged.
 
+== Metrics by Database
+
+{xref-cb-config}cache[cache] | {xref-cb-config}cbl_replication_pull[cbl_replication_pull] | {xref-cb-config}cbl_replication_push[cbl_replication_push] | {xref-cb-config}database[database] | {xref-cb-config}delta_sync[delta_sync] | {xref-cb-config}gsi_views[gsi_views] | {xref-cb-config}security[security] | {xref-cb-config}shared_bucket_import[shared_bucket_import]
+
 === per_db
 
 Stats for each database declared in the config file
```
Comment (simon-dew) [inline modules/ROOT/pages/stats-monitoring.adoc]: Make use of full stops consistent throughout.

File: modules/ROOT/pages/stats-monitoring.adoc [hunk 4]
```diff
@@ -232,291 +315,657 @@ Stats for each database declared in the config file
...
-Helps with channel cache tuning, and as a hint on cache size variation (when compared to average cache size).
+==== chan_cache_compact_time
+Description::
+Amount of time channel cache compaction has taken in total, across all compaction runs.
 
-===== chan_cache_misses
+==== chan_cache_hits
+Description::
+Channel cache requests fully served by the cache.
+Use Case::
```
Comment (simon-dew) [inline modules/ROOT/pages/stats-monitoring.adoc:519]: Make capitalisation consistent throughout. (`Use case` vs. `Use Case`)

**Review by simon-dew — CHANGES_REQUESTED** [review]
Just minor changes: markup, consistency of punctuation and capitalisation

**Comment by ibsoln** [comment]
Cheers, I’ll look at addressing those thing today

File: modules/ROOT/pages/stats-monitoring.adoc (hunk 1 above)
Comment (ibsoln) [reply modules/ROOT/pages/stats-monitoring.adoc]: Amended

File: modules/ROOT/pages/stats-monitoring.adoc (hunk 2 above)
Comment (ibsoln) [reply modules/ROOT/pages/stats-monitoring.adoc]: Change

File: modules/ROOT/pages/stats-monitoring.adoc (hunk 4 above)
Comment (ibsoln) [reply modules/ROOT/pages/stats-monitoring.adoc:519]: Changed to Use Case throughout

**Pull request merged**